docker-compose.yml

```diff
@@ -17,24 +17,52 @@ services:
     user: ${DOCKER_USER:-1000}
     image: joanie:development
     environment:
-      PYLINTHOME: /app/.pylint.d
+      - PYLINTHOME=/app/.pylint.d
+      - DJANGO_CONFIGURATION=Development
     env_file:
       - env.d/development/common
       - env.d/development/postgresql
     ports:
       - "8071:8000"
     volumes:
       - ./src/backend:/app
-      - ./data/static:/data/static
       - ./data/media:/data/media
     depends_on:
       - "postgresql"
+
+  app:
+    build:
+      context: .
+      target: production
+      args:
+        DOCKER_USER: ${DOCKER_USER:-1000}
+    user: ${DOCKER_USER:-1000}
+    image: joanie:production
+    environment:
+      - DJANGO_CONFIGURATION=ContinuousIntegration
+    env_file:
+      - env.d/development/common
+      - env.d/development/postgresql
+    volumes:
+      - ./data/media:/data/media
+    depends_on:
+      - postgresql
     networks:
       default:
       lms_outside:
         aliases:
           - joanie
 
+  nginx:
+    image: nginx:1.13
+    ports:
+      - "8081:8081"
```

Comment:
This port is already used within `richie-site-factory`. Maybe we should used another port to avoid conflict when these two projects are running at the same time?

Reply:
ok, 8082?